### DISCUSSION TITLE
Cannot login after login page redirect in a web view to a 404 and then nothing

### DISCUSSION THREAD

**Discussion by philipheinser** [community/community#34471] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.82.0 (79366424)
Device: iPhone 12 mini (iOS 16.0.2)
</details>

**Comment by HenSquared** [comment]
Hi @philipheinser, we are aware of this issue and are hoping to have a fix available very shortly. Sorry this has impacted your workflow, I will update once the problem is fixed. Thanks for your feedback!
